spinalcordtoolbox/utils.py

```diff
@@ -334,3 +334,4 @@ def _version_string():
 __sct_dir__ = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 __version__ = _version_string()
 __data_dir__ = os.path.join(__sct_dir__, 'data')
+__models_dir__ = os.path.join(__sct_dir__, 'models')
```

Comment:
This works, but it's a bit ad-hoc.

Reply:
what would be a better practice?

Reply:
You know what, I take that back. This is fine. This is exactly what `certifi` does:

```
$ ls
cacert.pem  core.py  __init__.py  __main__.py  __pycache__

$ cat core.py
...
def where():
    f = os.path.dirname(__file__)

    return os.path.join(f, "cacert.pem")
...
```

The Unix tradition I know puts associated asset/data files under /usr/share/$APP/, and the build/install script needs to tell the program(s) (usually in /usr/bin/$APP) that it has `DATADIR=/usr/share/$APP`.

For example, the video game 0 AD [has](https://www.archlinux.org/packages/community/x86_64/0ad/files)

<details>

```
    usr/
    usr/bin/
    usr/bin/0ad
    usr/bin/pyrogenesis
    usr/lib/
    usr/lib/0ad/
    usr/lib/0ad/libAtlasUI.so
    usr/lib/0ad/libCollada.so
    usr/lib/0ad/libmozjs38-ps-debug.so
    usr/lib/0ad/libmozjs38-ps-release.so
    usr/lib/0ad/libnvcore.so
    usr/lib/0ad/libnvimage.so
    usr/lib/0ad/libnvmath.so
    usr/lib/0ad/libnvtt.so
    usr/share/
    usr/share/0ad/
    usr/share/0ad/data/
    usr/share/0ad/data/l10n/
    usr/share/0ad/data/l10n/.tx/
    usr/share/0ad/data/l10n/.tx/config
    usr/share/0ad/data/l10n/ast.engine.po
    usr/share/0ad/data/l10n/bg.engine.po
    usr/share/0ad/data/l10n/ca.engine.po
    usr/share/0ad/data/l10n/cs.engine.po
    usr/share/0ad/data/l10n/de.engine.po
    usr/share/0ad/data/l10n/el.engine.po
    usr/share/0ad/data/l10n/en_GB.engine.po
    usr/share/0ad/data/l10n/engine.pot
    usr/share/0ad/data/l10n/es.engine.po
    usr/share/0ad/data/l10n/eu.engine.po
    usr/share/0ad/data/l10n/fr.engine.po
    usr/share/0ad/data/l10n/gd.engine.po
    usr/share/0ad/data/l10n/gl.engine.po
    usr/share/0ad/data/l10n/hu.engine.po
    usr/share/0ad/data/l10n/id.engine.po
    usr/share/0ad/data/l10n/it.engine.po
    usr/share/0ad/data/l10n/messages.json
    usr/share/0ad/data/l10n/ms.engine.po
    usr/share/0ad/data/l10n/nb.engine.po
    usr/share/0ad/data/l10n/nl.engine.po
    usr/share/0ad/data/l10n/pl.engine.po
    usr/share/0ad/data/l10n/pt_BR.engine.po
    usr/share/0ad/data/l10n/pt_PT.engine.po
    usr/share/0ad/data/l10n/ru.engine.po
    usr/share/0ad/data/l10n/sk.engine.po
    usr/share/0ad/data/l10n/sv.engine.po
    usr/share/0ad/data/l10n/tr.engine.po
    usr/share/0ad/data/l10n/uk.engine.po
    usr/share/applications/
    usr/share/applications/0ad.desktop
    usr/share/pixmaps/
    usr/share/pixmaps/0ad.png
```

</details>

It also depends on [`0ad-data`](https://www.archlinux.org/packages/community/any/0ad-data/files/) which has most of the actual game content:

```
usr/
usr/share/
usr/share/0ad/
usr/share/0ad/data/
usr/share/0ad/data/config/
usr/share/0ad/data/config/default.cfg
usr/share/0ad/data/config/keys.txt
usr/share/0ad/data/mods/
usr/share/0ad/data/mods/mod/
usr/share/0ad/data/mods/mod/mod.zip
usr/share/0ad/data/mods/public/
usr/share/0ad/data/mods/public/public.zip
...
```

The game [`wesnoth`](https://www.archlinux.org/packages/community/x86_64/wesnoth/files/) packages most of its content under

```
usr/share/wesnoth/data/ai
usr/share/wesnoth/data/ai/scenarios
usr/share/wesnoth/data/campaigns
```

Reply:
How do you feel about this though?

```suggestion
__models_dir__ = os.path.join(__data_dir__, 'models')
```

The `data/` dir already contains needed binaries, included some we even call 'models'.  The neural network models don't need special treatment.

I might even do:

```suggestion
__deepseg_dir__ = os.path.join(__sct_dir__, 'deepseg_models')
```

Reply:
makes sense-- fixed in e2f41e91be368886926f7a2f2b187d3819e7bc47